Tests/BotTests/XCTestManifests.swift

```diff
@@ -73,6 +74,8 @@ extension MergeServiceTests {
         ("test_pull_request_with_integration_label_and_conflicts", test_pull_request_with_integration_label_and_conflicts),
         ("test_pull_request_with_integration_label_and_ready_to_merge", test_pull_request_with_integration_label_and_ready_to_merge),
         ("test_pull_request_with_multiple_status_checks", test_pull_request_with_multiple_status_checks),
+        ("test_pull_request_with_non_required_failed_status_checks_requiresAllStatusChecks_off", test_pull_request_with_non_required_failed_status_checks_requiresAllStatusChecks_off),
+        ("test_pull_request_with_non_required_failed_status_checks_requiresAllStatusChecks_on", test_pull_request_with_non_required_failed_status_checks_requiresAllStatusChecks_on),
```

Comment:
Unrelated to this PR but it seems we forgot to update the TestManifest after the previous changes from last PR, so…